src/gengowatcher/secure_storage.py

```
@@ -0,0 +1,135 @@
+        self._cipher = Fernet(self._key)
+    
+    def _derive_key(self) -> bytes:
+        """Derive encryption key from system information"""
+        # Use a combination of system-specific information as salt
+        try:
+            # Try to get more system information
+            if hasattr(os, 'uname'):
+                salt_data = f"{os.getlogin()}_{os.uname().nodename}_{os.uname().machine}".encode()
+            else:
+                salt_data = f"{os.getlogin()}_{os.path.expanduser('~')}".encode()
+        except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)